### ISSUE TITLE
Project description is too wordy.

### ISSUE THREAD

**Issue by cassandracomar** [craftfrancis/gitStati#6]
Currently the project description reads:

```
 The intent of this system is to be a tool that can be used to gather information on the status of a specific or several specified repositories.
```

This is wordy enough to turn many people off especially because a lot of what's being said is redudant or assumed by context. The description should probably read something like:

```
A tool to gather information on the status of one or more specified repositories.
```

(The title at the top of the README should probably be the name of your project as well).

**Comment by meansa** [comment]
Upon review, we definitely see your point. Edited! (In our own words of course)
Thanks for the feedback.
